api/nakadi-event-bus-api.yaml

```diff
@@ -1245,6 +1245,115 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /timelines:
```

Comment:
I had only one point to make timeline api separate from event types: It is administrative api, which does not intersects with the event types api, that was an initial reason for that.